### PR TITLE
Update submodules

### LINKED ISSUE #19: Add mission file reader Subrepo

### PR DESCRIPTION
**Pull request by Philipp66904**
Note: This will not build as it requires a newer version of mission control in dev.

### CODE REVIEW

**Comment by Philipp66904** [comment]
Additionally, this PR will function as an update of the linked commits.
Therefore, it will not be merged before we have a working dev build.

**Comment by Philipp66904** [comment]
I accidentally pushed wrong branches for some of the submodules (interfaces for example is not pointing to `dev`).  
I will fix those before merging.

File: .gitmodules [hunk 1]
```diff
@@ -17,7 +17,7 @@
 [submodule "src/interfaces"]
 	path = src/interfaces
 	url = https://github.com/THI-Drone/interfaces.git
-	branch = dev
+	branch = feat-fcc-bridge-defs
```
Comment (J0hn-Lawrence) [inline .gitmodules]: Is this on purpose?

**Review by J0hn-Lawrence — COMMENTED** [review]
looks good to me, see my one comment

File: .gitmodules (hunk 1 above)
Comment (Philipp66904) [reply .gitmodules]: No, that's something that got introduced by a merge with the other branch

Comment (Philipp66904) [reply .gitmodules]: Fixed in https://github.com/THI-Drone/thi-drone-ws/pull/20/commits/a9b0ab32a7a65e40b82d5a2a8f422f351f3be836

**Comment by Philipp66904** [comment]
As soon as a working version of the FCC Bridge and Mission Control is available on dev, I will update this PR with the newest commits so that it can be used by V&V for their tests.

**Comment by J0hn-Lawrence** [comment]
> As soon as a working version of the FCC Bridge and Mission Control is available on dev, I will update this PR with the newest commits so that it can be used by V&V for their tests.

That's a good idea! Thanks!

**Review by J0las — DISMISSED** [review]
Franken (Frangen) gehört nicht zu Bayern.
Sieht auch @Philipp66904 so.
Lieber Fünfter als Fürther

**Review by J0las — APPROVED** [review]
Ich hab hunger, es ist spät.

**Review by J0hn-Lawrence — APPROVED** [review]
Approved, as we tested this in the lab.

**Pull request merged**